program/Project Descriptions and Plans/SDK/sdk-project-concept.md

```diff
@@ -13,20 +13,31 @@ The key objectives are:
 
-This Concept Proposal addresses version 1 of the SDK only. It will not include the Hardware Abstraction Layer (HAL), nor the CORE-V GNU compiler tool chain, since neither of these will be complete in time. There is no proposal to support the Clang/LLVM compiler tool chain.
+This Concept Proposal addresses version 1 of the SDK only. It will not include the Hardware Abstraction Layer (HAL), since this component, which will in turn require changes to the compiler tool chain, FreeRTOS kernel and FreeRTOS drivers, will not be ready in time.  We shall include the standard upstream GNU compiler tool chain, since the CORE-V specific GNU compiler tool chain is unlikely to be complete in time.  There is no proposal to support the Clang/LLVM compiler tool chain.
 
 ##  Components
 
-All the components which make up the SDK are shown in light green in the following image:
-![diagram showing the SDK components](./images/sdk-overview.png)
+The SDK runs on a _host_ computer (typically a PC running Windows, MacOS or Linux), connected via USB to the CORE-V MCU reference board, as shown in the following diagram:
 
-Note that some of the software potentially ends up on the board: libraries, HAL (not in version 1), FreeRTOS kernel and drivers.
+![diagram showing host connected to reference board](./images/devkit.jpg)
+
+The SDK consists of some components which run on the host computer, and some which are loaded on to the CORE-V MCU reference board.  The components which are on the host are shown in the following diagram:
+
+![diagram showing host sided SDK components](./images/sdk.jpg)
+
+The components which may be loaded on to the CORE-V MCU reference board include the HAL (although not in versions 1), the FreeRTOS kernel and drivers, the compiler and operating system libraries, and the applications, which may be derived from the examples within the SDK.  These are shown in the following diagram:
+
+![diagram showing reference board  SDK components](./images/refbrdsw.jpg)
 
 See the [Summary of requirements](#summary-of-requirements) for the requirements driving each of these components.
 
 ### Component 1 - OpenHW IDE and debugger
 
-This is based on Eclipse CDT
+This is based on Eclipse CDT.  It is divided into two sub-components:
+
+* The **IDE** component comprising  "ready-to-launch" application examples for the list of supported boards, tool chains, debuggers, and libraries; and
+
+* The **Debugger** component, which will supply corresponding binaries plus connection and initialization parameters.
 
 **Current status:** Initial development by Alexander Fedorov operational, with minimal examples and using the upstream compiler tool chain. Debugger integration for the development board will be required, and cannot be started until this is fully specified, nor tested until hardware is available. Symbiflow tooling (see [Component 5](#component-5---symbiflow-tooling)) support will require additional work
```

Comment:
It needs to be decided which reference boards we want to support for "version 1" of SDK. For each board we will need:

* creation entry point (wizard)
* project templates
* toolchain setup
* libraries setup
* launch and debug setup
* context help
* application samples

Reply:
I've made it clear that the target board is the final release board, and any prototype is only useful so far as it matches the final board.